### DISCUSSION TITLE
據說每台電腦只能試一次

### DISCUSSION THREAD

**Discussion by BeaGle0928** [1101-GS4719-C/Python-Community-GS4719-C#1748] [General]
試試看把電腦解析度降到800*600，再把UI放大到500%，到底會發生什麼事呢？如果你已經試了的話這裡有以下解決方法，不保證適用。

方法1.
無副螢幕者按下鍵盤上的開始鏈後，直接輸入"解析度"再按Enter鍵兩下，此時電腦會自動開啟設定，並將選項框框到解析度選項上，接著再按下一次Enter鍵，則會看到
800X600的解析度選項，此時使用鍵盤方向
鍵將選項往上選至1920X1080解析度後按下
Enter鍵並保留設定，即可將解析度調回至
FHD。接著可直接使用滑鼠滾輪將設定畫面
往下滾，可以看到"關閉自訂縮放比例並登出"，選取該選項即可將電腦調回至正常。
(如果是筆電用戶且無滑鼠者，調完解析度後
可以使用tab鍵連按，連按至黑色選取框選
到"關閉自訂縮放比例並登出"，即可將電腦
回復至正常）
方法2.
有其他副螢幕者將副螢幕接上後通常會自動
偵測解析度，並顯示1080p，此時用滑鼠點
擊桌面右鍵並選顯示其他選項，即可看到顯示設定，點擊進去後再找到關閉自訂縮放比例並登出，即可將電腦調回至正常

我想應該連微軟自己可能都沒有想過有人會想出這個bug...

資料來源：https://youtube.com/shorts/TESgOkDhcQY?si=h3pX_riMYmH3J0CX

**Comment by davidteng00000** [comment]
解決這個問題只需要外接一個螢幕呀...

**Comment by chiuwangwang** [comment]
e04，我拿舊筆電去試，真的壞掉了。我不信邪，我就是手賤。@BaeGle0928，陪我一台筆電，或著折現也可以，等你回復。
